solr/core/build.gradle

```diff
@@ -149,5 +149,12 @@ dependencies {
   testImplementation('org.mockito:mockito-core', {
     exclude group: "net.bytebuddy", module: "byte-buddy-agent"
   })
+  testImplementation('no.nav.security:mock-oauth2-server', {
```

Comment:
I add this as a testImpl dependency, but still I'm forced to add LICENSE and NOTICE files in the `licenses` folder for this and all its dependencies. Why is that necessary, when we do not distribute these jars, they are only used during tests?
There are the new (test) deps:
```
+--- no.nav.security:mock-oauth2-server -> 0.3.3
|    +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.32
|    |    +--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.32
|    |    \--- org.jetbrains:annotations:13.0
|    +--- org.jetbrains.kotlin:kotlin-reflect:1.4.32
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.4.32 (*)
|    +--- com.fasterxml.jackson.core:jackson-databind:2.12.2 (*)
|    +--- io.github.microutils:kotlin-logging:2.0.6
|    |    \--- io.github.microutils:kotlin-logging-jvm:2.0.6
|    |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.31 -> 1.4.32 (*)
|    |         \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.31 -> 1.4.32
|    +--- com.fasterxml.jackson.module:jackson-module-kotlin:2.12.2
|    |    +--- com.fasterxml.jackson.core:jackson-databind:2.12.2 (*)
|    |    +--- com.fasterxml.jackson.core:jackson-annotations:2.12.2 (*)
|    |    +--- org.jetbrains.kotlin:kotlin-reflect:1.4.21 -> 1.4.32 (*)
|    |    \--- com.fasterxml.jackson:jackson-bom:2.12.2 (*)
|    +--- org.freemarker:freemarker:2.3.31
|    +--- com.squareup.okhttp3:mockwebserver:4.9.1
|    |    +--- com.squareup.okhttp3:okhttp:4.9.1
|    |    |    +--- com.squareup.okio:okio:2.8.0
|    |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.0 -> 1.4.32 (*)
|    |    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib-common:1.4.0 -> 1.4.32
|    |    |    \--- org.jetbrains.kotlin:kotlin-stdlib:1.4.10 -> 1.4.32 (*)
|    |    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.10
|    |         +--- org.jetbrains.kotlin:kotlin-stdlib:1.4.10 -> 1.4.32 (*)
|    |         \--- org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.4.10
|    |              \--- org.jetbrains.kotlin:kotlin-stdlib:1.4.10 -> 1.4.32 (*)
|    \--- com.nimbusds:oauth2-oidc-sdk:9.2.3
|         +--- com.github.stephenc.jcip:jcip-annotations:1.0-1
|         +--- com.nimbusds:content-type:2.1
|         +--- net.minidev:json-smart:[1.3.1,2.3] -> 2.3
|         |    \--- net.minidev:accessors-smart:1.2
|         |         \--- org.ow2.asm:asm:5.0.4 -> 7.2
|         +--- com.nimbusds:lang-tag:1.4.4
|         \--- com.nimbusds:nimbus-jose-jwt:9.6.1
|              \--- com.github.stephenc.jcip:jcip-annotations:1.0-1
```

Reply:
Anyone knows about this?

Reply:
See https://issues.apache.org/jira/browse/SOLR-15465 for a separate discussion of this topic, leaving it in for now.